src/insexpand.c

```diff
@@ -4330,17 +4367,26 @@ ins_compl_delete(void)
 /*
  * Insert the new text being completed.
  * "in_compl_func" is TRUE when called from complete_check().
+ * "move_cursor" is used when completeopt include preinsert and when true
+ * mean accept from edit function
```

Comment:
1. Doesn't `edit()` call this with `move_cursor == FALSE`?
2. The "edit function" part is meaningless. The whole Insert mode main loop is the "edit function".